ApiDoctor.Validation/TableSpec/tablespecconverter.cs

```diff
@@ -84,8 +84,8 @@ public TableDefinition ParseTableSpec(Block tableSpecBlock, Stack<Config.Documen
 
             TableDecoder decoder = new TableDecoder { Type = TableBlockType.Unknown };
 
-            var headerText = headerStack.Peek()?.Title;
-            
+            var headerText = headerStack.Count > 0 ? headerStack.Peek()?.Title : null;
```

Comment:
this looks equivalent to what it was doing before. why the change?

Reply:
Would throw an exception  if Peek() was called and the stack was empty.